filebeat/prospector/log/file.go

```diff
@@ -146,25 +146,25 @@ func (r *LogFile) errorChecks(err error) error {
 	return nil
 }
 
-func (r *LogFile) wait() {
+func (f *File) wait() {
 
 	// Wait before trying to read file again. File reached EOF.
 	select {
-	case <-r.done:
+	case <-f.done:
 		return
-	case <-time.After(r.backoff):
+	case <-time.After(f.backoff):
 	}
 
 	// Increment backoff up to maxBackoff
-	if r.backoff < r.config.MaxBackoff {
-		r.backoff = r.backoff * time.Duration(r.config.BackoffFactor)
-		if r.backoff > r.config.MaxBackoff {
-			r.backoff = r.config.MaxBackoff
+	if f.backoff < f.config.MaxBackoff {
+		f.backoff = f.backoff * time.Duration(f.config.BackoffFactor)
+		if f.backoff > f.config.MaxBackoff {
+			f.backoff = f.config.MaxBackoff
 		}
 	}
 }
 
-func (r *LogFile) Close() {
-	close(r.done)
+func (f *File) Close() {
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported method File.Close should have comment or be unexported